src/ducks/billsMatching/Linker/billsToOperation/index.js

```diff
@@ -0,0 +1,30 @@
+const { operationsFilters } = require('./operationsFilters')
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.